skimage/_build.py

```diff
@@ -24,6 +24,11 @@ def cython(pyx_files, working_path=''):
     if len(sys.argv) >= 2 and sys.argv[1] == 'clean':
         return
 
+    cython_options = {}
+    if os.environ.get('CACHE_CYTHON', 'False').lower() == 'true':
```

Comment:
Hmm... OK. `ccache` doesn't seem to be having an affect either. I was hoping that the Cython caching might help.